tests/func/test_scm.py

```diff
@@ -44,3 +46,76 @@ def test_lfs_prefetch(tmp_dir, dvc, scm, mocker):
     with dvc.switch(rev):
         lfs_prefetch(dvc.dvcfs, ["foo"])
         mock_fetch.assert_called_once()
+
+
+def test_download_lfs_prefetch_directory(tmp_dir, dvc, scm, mocker):
```

Comment:
should these test belong to something like `test_download.py`?

Reply:
Sounds good. I wasn't sure what the test structure under `tests/func/` is. I've moved the tests as you suggested.